config/NuElectronPXSec.xml

```diff
@@ -14,8 +14,10 @@ WeinbergAngle  double   No                                 CommonParam[WeakInt]
 
   <param_set name="Default"> 
     <param type="string"  name="CommonParam"> WeakInt  </param>
-    
+    <param type="int"    name ="N-Integration-Samples" >              2000  </param>
      <param type="alg"  name="XSec-Integrator"> genie::NuElectronXSec/Default  </param>
+     <param type="alg"  name="Bohr-Velocity"> genie::BohrElectronVelocity/Default  </param>
```

Comment:
No, we only need one entry, which has name "ElectronVelocity", It will be the user to decide which velocity to set.

Reply:
In the default add the Bohr velocity. 
Then create a "Static" parameter set in which the number of integration samples is 1 and the electron velocity is the static one